README.md

```diff
@@ -1,66 +1,61 @@
 
-#### <a name="InstallingAPK">Option 1: Install the prebuilt WebARonARCore APK</a>
+...or by using ADB:
 
-We have provided a prebuilt APK for your convenience inside this repo. You can install it:
+* Download the [WebARonARCore APK](https://github.com/google-ar/WebARonARCore/blob/webarcore_57.0.2987.5/apk/WebARonARCore.apk). 
+* Install the APK to your device:
+  * `$ adb install -r /path/to/WebARonARCore.apk`
 
-##### Directly from a device
+Alternatively, the WebARonARCore APK can be [built and installed from source](#BuildingFromSource).
 
-* Open an web browser on your Android device.
-* Open this page and click [here](https://github.com/google-ar/WebARonARCore/raw/webarcore_57.0.2987.5/apk/WebARonARCore.apk) to download and install the WebARonARCore APK directly.
-* Launch the WebARonARCore app from your device.
+### <a name="ViewingExamples">3. Viewing examples</a>
+A [list of example scenes](https://developers.google.com/ar/develop/web/getting-started#examples) compatible with WebARonARCore and [WebARonARKit](https://github.com/google-ar/WebARonARKit) are available at [developers.google.com](https://developers.google.com/ar/develop/web/getting-started#examples).
 
-##### Using ADB
+### <a name="BuildingScenes">4. Building your own scenes</a>
+To build AR web experiences that work with WebARonARCore (or [WebARonARKit for iOS](https://github.com/google-ar/WebARonARKit)), we recommend **[three.ar.js](https://github.com/google-ar/three.ar.js)**, a helper library that works with the popular [three.js](http://threejs.org) WebGL framework. [Three.ar.js](https://github.com/google-ar/three.ar.js) provides common AR building blocks, such as a visible reticle that draws on top of real world surfaces, and [example scenes](https://github.com/google-ar/three.ar.js#examples).
 
-* Download the WebARonARCore APK from [here](https://github.com/google-ar/WebARonARCore/blob/webarcore_57.0.2987.5/apk/WebARonARCore.apk). 
-* Install the WebARonARCore APK to your device: `$ adb install -r WebARonARCore.apk`
-* Launch the WebARonARCore app from your device.
 
-#### <a name="CompileFromSource">Option 2: Build the WebARonARCore APK from source</a>
+## <a name="BuildingFromSource">Building the WebARonARCore APK from source</a>
 
-##### Clone the git repo and prepare it to be built
-
-Instructions for [cloning and building Chromium](https://www.chromium.org/developers/how-tos/android-build-instructions) are available at [chromium.org](https://www.chromium.org/developers/how-tos/android-build-instructions)
+WebARonARCore can optionally be built and installed from source. First, clone the git repo and prepare it to be built. Instructions for [cloning and building Chromium](https://www.chromium.org/developers/how-tos/android-build-instructions) are available at [chromium.org](https://www.chromium.org/developers/how-tos/android-build-instructions)
```

Comment:
I would avoid the sentence "first, clone the git repo and prepare it to be built" as it is misleading. Which repo, this one? Chromium repo? Does not add much info and can lead to mistakes IMHO.